packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -51,6 +55,18 @@ import IImageCalibration from '../types/IImageCalibration';
  * logic.
  */
 class Viewport implements IViewport {
+  static readonly CameraViewPresentation: ViewPresentationSelector = {
+    rotation: true,
+    pan: true,
+    zoom: true,
+    displayArea: true,
+  };
+
+  static readonly TransferViewPresentation: ViewPresentationSelector = {
```

Comment:
requires doc on what is TransferViewPresentation, and how is it different from CameraViewPresentation

Reply:
I don't see getting used anywhere